widgetssdk/src/main/java/com/glia/widgets/call/CallView.kt

```diff
@@ -998,6 +992,27 @@ internal class CallView(
         }
     }
 
+    private fun setVisibilityBasedOnViewState(view1: View, view2: View, state: ViewState) {
```

Comment:
I would suggest to avoid names like `view1`, `view2`. Here, for example, we can name them `button` and `label`, I think.

Reply:
I added Davids more elegant solution